hw18/api/api.spec.ts

```diff
@@ -0,0 +1,132 @@
+ import superagent from "superagent";
+ import { describe, expect } from "@jest/globals";
```

Comment:
There is no need to import this modules, as `describe` and `expect` are globally declared.